moodle/Dockerfile

```diff
@@ -5,7 +5,7 @@ ARG LOCAL_CODECHECKER_VERSION=v5.0.0
 ARG LOCAL_MOODLECHECK_VERSION=v1.1.8
 ARG TOOL_PLUGINSKEL_VERSION=1.5.3
 ARG TOOL_CERTIFICATE_VERSION=v4.2
-ARG LOCAL_RAISECLI_VERSION=2023061400
+ARG LOCAL_RAISECLI_VERSION=2023103100
```

Comment:
This should be `2023103100-prerelease` to match the tag you pushed (otherwise the Docker build fails when trying to clone the code).